js/layout-javascript/agenda-code.js

```diff
@@ -2547,7 +2569,38 @@ DynamicList.prototype.addDetailViewData = function(entry) {
       content = entry.originalData[dynamicDataObj.column];
     }
 
-    content = _this.Utils.String.toFormattedString(content);
+    if (dynamicDataObj.type === 'image') {
```

Comment:
Everything inside this `if` clause seems to be repeated across all layouts too. You only made the regex part DRY with the `getImagesUrlsByRegex` function but I feel that there's a lot more that can be modularised.